backend/lib/calculate/adequacy.py

```diff
@@ -162,6 +155,10 @@ def calculate_adequacies(
 
     locations = list(location_mapping.keys())
 
+    logger.debug(
+        'Calculating adequacies for {} provider locations and {} service areas using {}.'.format(
+            len(locations), len(service_area_ids), measurer_name))
```

Comment:
Maybe use `len(location_mapping)` to not log the duplicates? Or log this information as well as `x unique locations`?